display.py

```diff
@@ -5,14 +5,14 @@
 
 import time
 
-name=raw_input("Enter Your Name : ")
+name=raw_input("Enter Your Name : )
 
 #the width of the display
 #(the windows console is 79 characters wide).
 WIDTH = 79
 
 #the message we wish to print
-message = name.upper()
+message = name.upper(
```

Comment:
Add a missing closing parenthesis for the `upper` method to avoid syntax errors.